src/AppInstallerCommonCore/Public/AppInstallerMsixManifest.h

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+#include <AppxPackaging.h>
+#include <AppInstallerStrings.h>
+#include <AppInstallerVersions.h>
+#include <wrl/client.h>
+
+#include <string>
+
+namespace AppInstaller::Msix
+{
+    using namespace Microsoft::WRL;
+    using string_t = Utility::NormalizedString;
+
+    namespace Platform
+    {
+        constexpr auto& WindowsDesktop = "windows.desktop";
+        constexpr auto& WindowsUniversal = "windows.universal";
+
+        bool IsWindowsDesktop(std::string platformName);
+        bool IsWindowsUniversal(std::string platformName);
+    }
+
+    // Four parts version number: 16-bits.16-bits.16-bits.16-bits
+    struct FourPartsVersionNumber : public Utility::Version
+    {
+        FourPartsVersionNumber() = default;
+        FourPartsVersionNumber(std::string version) : Version(version) {}
```

Comment:
```suggestion
        FourPartVersionNumber(std::string version) : Version(std::move(version)) {}
```

Should it also verify that there are no more than 4 parts, and none of them have the "other" field?
And it should be with `Version`, as it isn't MSIX specific.

Reply:
Yes, good point. Updated implementation and moved to `Version`.